Luke7787/ImmaculateInventors
# Feature/HomePage

**Summary:**

Implemented a visually appealing homepage design that enhances user engagement.
- Enhanced homepage with a striking header image and blur effect.
- Added a "TRY NOW" button for direct navigation to the inventory page.
- "Key Features" section showcasing product functionalities.
- "Try Our Product" subsection with a placeholder for future content.

**How to test: npm start**

- "TRY NOW" works - Tested ✓
- "Key Features" section [ images and texts align correctly ] - Tested ✓
- "Try Our Product" section is displayed properly at the bottom - Tested ✓

**Note: "Try Our Product" subsection is not implemented yet**
